### ISSUE TITLE
script 

### ISSUE THREAD

**Issue by BDXH-FY24** [Bioinformatics-Service-MPI-Munster/stewen-2024-nat-comms#1]
Hi Stewen,

Can you share the script for rhapsody-demultiplex and rhapsody-extract-barcode?

**Comment by kaukrise** [comment]
They are available as part of a more general Python module: https://github.com/Bioinformatics-Service-MPI-Munster/scrna-tools

You can install it with `pip`.

**Comment by BDXH-FY24** [comment]
pip install scrna-tools?

**Comment by kaukrise** [comment]
No, it is not available on Pypi, you will have to `git clone` the repository first and then install from there. Here are the basic steps:

```
git clone https://github.com/Bioinformatics-Service-MPI-Munster/anndataview
git clone https://github.com/Bioinformatics-Service-MPI-Munster/scrna-tools
```

And then install with `pip`:

```
pip install ./anndataview
pip install ./scrna-tools
```

I've added these instructions to the README, too.

**Comment by BDXH-FY24** [comment]
Thanks, that works.

**Comment by BDXH-FY24** [comment]
one more ask, could you please share the file: rhapsody_whitelist.txt

**Comment by kaukrise** [comment]
You can find it here, at the bottom of the page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE223738

**Comment by BDXH-FY24** [comment]
cool, thank you so much!

**Comment by BDXH-FY24** [comment]
One more question, the beads for Rhapsody is enhance V2?

**Comment by kaukrise** [comment]
No, these are V1

**Comment by BDXH-FY24** [comment]
Thanks a lot
